automations/js/src/project_automation/issues.mjs

```diff
@@ -1,5 +1,7 @@
 import { getBoard } from '../utils/projects.mjs'
 
+import { debug } from '@actions/core'
+
```

Comment:
We cannot directly import from `@actions/core` here. Explained in the other comment.